docker-compose.yaml

```diff
@@ -17,7 +17,7 @@ services:
 
   deploy-sequencer-contracts:
     image: ghcr.io/espressosystems/espresso-sequencer/deploy:main
-    command: deploy --only fee-contract
+    command: deploy --only sequencer,permissioned-stake-table
```

Comment:
what is the `sequencer` contract here? I only see fee, LC, and stake table contracts

Reply:
I might have edited this by accident because I was messing with contract groups some time ago.